index.js

```diff
@@ -23,10 +23,16 @@ try {
   // Fetch inputs
   const name = core.getInput('name');
   const environment = core.getInput('environment');
+  const domain = core.getInput('domain');
+  const output = branchName(github.context).toLowerCase().replace(/[^a-z0-9-]/g, "").substr(0, 63)
 
   const deploy = deployName(name, environment, github.context)
   console.log(`name: ${deploy}`);
   core.setOutput('name', deploy);
+
+  const url = `${deploy}.${domain}`
+  core.setOutput('url', url);
+  core.setOutput('branch', output);
```

Comment:
Follow same pattern as above with other outputs - `console.log` value and then set output. Makes life much easier when you can see in logs what this action actually outputs.

Reply:
https://github.com/itsmycargo/action-deploy-environment/pull/1/commits/5a65b1686611b04658413151dfb0f9b926538763